### ISSUE TITLE
Hosting the Health Check routes in an existing Akka HTTP server

### ISSUE THREAD

**Issue by sdeigm** [akka/akka-management#599]
Hosting the Akka HTTP managment routes in an existing HTTP server currently only provides access to the routes dealing with the the cluster membership and status, not the Health Check routes.

It would be good to also include the Health Check routes (readiness and liveness) into this interface so that they can be reached at the same port address as the other routes. This avoids the opening of another HTTP server port for just that reason.

**Comment by TimMoore** [comment]
It looks to me like `AkkaManagement(system).routes` will give you all of the configured routes, including health checks.

https://github.com/akka/akka-management/blob/1db736af3661fd98af299dd8bfc0dbbccff2900d/management/src/main/scala/akka/management/scaladsl/AkkaManagement.scala#L88-L95

@sdeigm could you try that and report back?

@akka/akka-team, should that be the documented approach?

https://doc.akka.io/docs/akka-management/current/cluster-http-management.html#hosting-the-routes-in-an-existing-akka-http-server

**Comment by johanandren** [comment]
There is one difference though and that is that it potentially adds modifying routes, join cluster, down node etc. While health check alone would perhaps be less dangerous.

**Comment by TimMoore** [comment]
Good point. It would definitely be useful to be able to get hold of the health check routes on their own. My interpretation of the original post was that @sdeigm wanted both anyway, but maybe not.

**Comment by sdeigm** [comment]
> @sdeigm could you try that and report back?

I just tried it out and it works. A /alive and a /ready route are added to my server. Thanks for the tip!

And yes you are correct @TimMoore, in my case I want the routes to modify the cluster membership anyway. So for me personally, this solves the issue, but for others that might be a problem.

**Comment by TimMoore** [comment]
Thanks for confirming @sdeigm ... this should at least be documented somewhere, IMHO, so maybe we can leave the issue open for that?

**Comment by TimMoore** [comment]
A request to allow embedding _only_ the health check routes in an existing server is in #605.

**Comment by chbatey** [comment]
> Thanks for confirming @sdeigm ... this should at least be documented somewhere, IMHO, so maybe we can leave the issue open for that?

Leaving open for improving the docs
